setup.py

```diff
@@ -18,7 +18,7 @@ def readme():
 
 base_requirements = [
     'attrs~=17.4',
-    'conformity~=1.13',
+    'conformity~=1.17.0',
```

Comment:
Ah, man, I'm sorry. This was my bad. `'1.17.0'` is too specific for a library. Should be `'1.17'`.